server/routes/support/handlers/offenderDetail.ts

```diff
@@ -78,7 +78,7 @@ export default class OffenderDetailRoutes {
         name: probationPractitioner
           ? `${probationPractitioner.staff.forenames} ${probationPractitioner.staff.surname}`
           : '',
-        staffCode: probationPractitionerContact.staffCode,
+        staffCode: probationPractitionerContact?.staffCode,
```

Comment:
Fix issue with showing error page when no COM is assigned - it's a bit ugly ATM as it just shows a load of empty fields but better than blowing up